app/assets/stylesheets/_global.scss

```diff
@@ -9,5 +9,40 @@ body {
 }
 
 .nav-bar {
-  position: relative;
+  margin-top: .75em;
+  text-align: right;
+}
+
+#page-header {
+  background-color: #f8f8f8;
+  background-image: -webkit-linear-gradient(top, white 0%, #f8f8f8 100%);
+  background-image: -o-linear-gradient(top, white 0%, #f8f8f8 100%);
+  background-image: linear-gradient(to bottom, white 0%, #f8f8f8 100%);
+  background-repeat: repeat-x;
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#FFFFFFFF', endColorstr='#FFF8F8F8', GradientType=0);
+  filter: progid:DXImageTransform.Microsoft.gradient(enabled = false);
```

Comment:
Let's add bourbon to the project, then all of the above can be replaced with a [linear-gradient](http://bourbon.io/docs/#linear-gradient)

```scss
@include linear-gradient(top, white 0%, #f8f8f8 100%);
```